plugins/vultr/vultr-cli.go

```diff
@@ -0,0 +1,22 @@
+package vultr
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/needsauth"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+)
+
+func VultrCLI() schema.Executable {
+	return schema.Executable{
+		Name:      "Vultr CLI",
+		Runs:      []string{"vultr-cli"},
+		DocsURL:   sdk.URL("https://github.com/vultr/vultr-cli"),
+		NeedsAuth: needsauth.NotForHelpOrVersion(),
```

Comment:
Let's opt out of authentication when `--config` is specified, since this indicated a custom credential being provisioned.

Reply:
That's definitely something I considered but for some reason assumed that users would use the same API Key both on environment variable and via config file. I have now updated the plugin to not use 1Password-stored API Key for `--config` commands.

Reply:
Thank you for addressing this. Could you add both these conditions to the authentication criteria? You can do this using `IfAll`. Something along the lines of:
```
NeedsAuth: needsauth.IfAll(
  needsauth.NotForHelpOrVersion(),
  needsauth.NotWhenContainsArgs("--config"),
)
```

Additionally, should this plugin prompt for authorization when no arguments are provided? i.e. only calling `vultr-cli`, with no other subcommands/flags/args.

Reply:
I feel silly removing the NotForHelpOrVersion() check. 😅 All changes should be addressed now: https://github.com/1Password/shell-plugins/pull/159/commits/acecc5d8ed77b3a456f4891e1ab945b8c7567086. 

Thanks for all the guidance, @hculea!